packages/macro/src/babel-plugin-macros.d.ts

```diff
@@ -0,0 +1,7 @@
+import * as _babelPluginMacros from "babel-plugin-macros";
+
+declare module "babel-plugin-macros" {
+  interface MacroParams {
+    config: any;
+  }
+}
```

Comment:
The types for babel-plugin-macros are missing typings for the `config` argument, so I've had to workaround that via module augmentation. We can submit a PR to DefinitelyTyped to get these types updated.